client/src/components/AddNewItem/AddNewItem.tsx

```diff
@@ -0,0 +1,46 @@
+import { Box, Button, Grid, MenuItem, Select, Typography } from '@material-ui/core';
+import Input from '@material-ui/core/Input';
+import * as React from 'react';
+import useStyles from './useStyles';
+
+export default function AddNewItem(): JSX.Element {
+  const classes = useStyles();
+  const [list, setList] = React.useState('');
+  const [lists] = React.useState(['Clothes', 'Furniture', 'Luxury']);
```

Comment:
It makes more sense to mimick what is actually going to happen with this. We are going to fetch lists on the dashboard, and pass it down to `AddNewItem` as a prop.

Reply:
fixed